src/lib/shared/api/proxies.ts

```diff
@@ -0,0 +1,18 @@
+import type { SBEpisode, Episode, SBPodcast, Podcast } from './types';
```

Comment:
issue: the file name is kind of confusing to me. These aren't really proxies, I'd call them _adapters_